src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalAmount = 0;
+  const znyzhka = 20;
+  const vznyzhka = 50;
```

Comment:
The variable names `znyzhka` and `vznyzhka` are not descriptive in English. Consider renaming them to `smallDiscount` and `largeDiscount` for better readability.